### DISCUSSION TITLE
GitHub has become too stressful and demanding for hobbyist programmers

### DISCUSSION THREAD

**Discussion by Tatwi** [community/community#43862] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

TL;DR: The enormous focus on activity tracking and how absolutely **IN YOUR FACE** it is, everywhere, completely does not jibe with my use of this service. In fact, it is so stressful that I have avoided working on my projects entirely. **The contribution calendar genuinely gives me heartburn and causes long term anxiety/depression.**

I understand that GitHub now belongs to a huge corporation that is focused on productivity, but this service used to be welcoming to we individual, every-day people who do not care about such things. As such, we should have the option to disable the tracking and display of our activity. In fact, we should be have the option to disable a number of things. It's not our fault Microsoft bought GitHub.

Things on GitHub that I want to disable:

- All the contribution and activity tracking and all the charts displayed on the main (profile) page.
- Achievements!
- The Projects thing.
- The Insights thing.
- The Wiki thing. I have always been amazing at writing and including documentation within my content. I don't want to feel I am supposed to use your proprietary thing too.
- The Security thing. I don't care about computer security. It's an a non-subject. After 30+ years, I'm sick of hearing about it. As basic user of GitHub, I certainly don't give any wingdings whatsoever about it in relation to my repos. If someone wants to write me an email about an issue, cool, I will read it, but I certainly don't need an entire framework to manage this one stupid topic. You already have the whole "Issues" thing for tracking... issues.

I don't want to move to another service, because I already have this one setup and I've been using it for the better part of a decade. All I really want from GitHub is...

1. Remote mirroring/hosting of my repos. 
2. GitHub Pages are genuinely useful and appreciated.

That's it. I would happily disable everything else, including this community thing we appear to be forced to use just to provide feedback on how we use GitHub.

**Comment by rosedalerk** [comment]
You are right that GitHub has become too stressful and demanding for hobbyist programmers. However, achievements, projects, and security are good to have.

**Comment by sryze** [comment]
I would also like to be able to hide the activity timeline. Not just because of the reasons mentioned, but also because of privacy concerns. I don't feel ok having my entire GitHub activity for the last year graphed by day of week or month in my profile for everyone to see. Having a just the linear history would be less concerning for me.
